spec/models/early_career_payments/admin_tasks_presenter_spec.rb

```diff
@@ -5,14 +5,16 @@
   let(:eligibility) { claim.eligibility }
 
   let(:claim) do
-    build(:claim,
-      academic_year: "2019/2020",
-      eligibility: build(:maths_and_physics_eligibility,
-        teaching_maths_or_physics: true,
+    build(
+      :claim,
+      academic_year: AcademicYear.new(2021),
+      eligibility: build(
+        :early_career_payments_eligibility,
         current_school: school,
-        initial_teacher_training_subject: :maths,
-        initial_teacher_training_subject_specialism: nil,
-        qts_award_year: "on_or_after_cut_off_date"))
+        eligible_itt_subject: :mathematics,
```

Comment:
Not sure about these types.

Reply:
@nomasprime Are you asking for an opinion or help with this? 
I am unsure if the comment is here for you as a reminder to review and then decide, or if you are asking for help.

I can see that lines 13-15 that are deleted were not appropriate or associated to an ECP, as they are attributes for M+P.

Reply:
Thought I needed serialized types at DB level and cast otherwise but not sure this is the case, mabye Rails is doing some magic.

Reply:
No magic, should be serialized value. If it works otherwise, it's coincidental.